pkg/scheduler/server/api.go

```diff
@@ -39,6 +39,13 @@ func (s *Server) ScheduleJob(ctx context.Context, req *schedulerv1pb.ScheduleJob
 		return nil, err
 	}
 
+	_, err = cron.Get(ctx, serialized.Name())
```

Comment:
Hey @WhitWaldo thanks for the PR and initiative here 🎉 

Did you by chance see [this PR](https://github.com/diagridio/go-etcd-cron/pull/74/files)? My understanding was that we will want to expose to users the option to simply `Add` or `AddIfNotExists` following the function names, basically overwrite the job or not. They should be able to toggle it, then we will want to call the functions exposed from that PR instead of doing a `Get` call.